pcsx2/gui/Panels/CpuPanel.cpp

```diff
@@ -293,9 +293,6 @@ void Panels::CpuPanelVU::AppStatusEvent_OnSettingsApplied()
 
 void Panels::CpuPanelVU::ApplyConfigToGui( AppConfig& configToApply, int flags )
 {
-	m_panel_VU0->Enable(true);
-	m_panel_VU1->Enable(true);
-
 	m_panel_VU0->EnableItem( 1, true);
 #ifndef DISABLE_SVU
 	m_panel_VU0->EnableItem( 2, true);
```

Comment:
Since it seems enabled by default (unless I'm missing a test case), it might be better to remove the first `EnableItem()` and change the remaining lines to:
```
#ifdef DISABLE_SVU
	m_panel_VU0->EnableItem(2, false);
#endif
```
Same applies below.